notebooks/random_forest_clean.ipynb

```diff
@@ -0,0 +1,442 @@
+{
```

Comment:
<p>rg_scaled not defined? Stuff like this should get tidied up before we merge to master.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/srensi/tmprss2/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>